include/z64eff_footmark.h

```diff
@@ -0,0 +1,33 @@
+#ifndef Z64EFF_FOOTMARK_H
+#define Z64EFF_FOOTMARK_H
+
+#include "ultra64.h"
+#include "z64math.h"
+
+struct PlayState;
+struct Actor;
+
+typedef struct EffFootmark {
+    /* 0x00 */ MtxF displayMatrix;
+    /* 0x40 */ struct Actor* actor;
+    /* 0x44 */ Vec3f position;
```

Comment:
```suggestion
    /* 0x44 */ Vec3f pos;
```
I think we typically keep position shortened like this?

Reply:
in this same vein its pretty common to abbreviate MtxF's as just mf, but I didnt comment on it since it wasnt a part of these changes

Reply:
Let's rename that one too then lol